src/Microsoft.Health.Fhir.Liquid.Converter.Tool/ConverterLogicHandler.cs

```diff
@@ -110,6 +111,7 @@ private static ITemplateProvider CreateTemplateProvider(DataType dataType, strin
                 DataType.Hl7v2 => new TemplateProvider(templateDirectory, DataType.Hl7v2),
                 DataType.Ccda => new TemplateProvider(templateDirectory, DataType.Ccda),
                 DataType.Json => new TemplateProvider(templateDirectory, DataType.Json),
+                DataType.Fhir => new TemplateProvider(templateDirectory, DataType.Fhir),
                 _ => throw new NotImplementedException($"The conversion from data type {dataType} to FHIR is not supported")
```

Comment:
maybe we can remove this switch, just use new TemplateProvider(templateDirectory, dataType)?

Reply:
Thanks, removed switch